media-gfx/viewnior/viewnior-1.8.ebuild

```diff
@@ -16,7 +16,7 @@ IUSE=""
 
 RDEPEND="
 	dev-libs/glib:2
-	media-gfx/exiv2:0=
+	<media-gfx/exiv2-0.28.0:0=
```

Comment:
This would technically require a revbump, and upper version bounds make terrible UX. My advice:
- exiv2-0.28.0 is PMASKED, so there's no hurry
- leave the stable ebuild alone, `~arch/arch` mixing is unsupported (and more responsibility on the user)
- make sure to file stabilisation bug in time before exiv2 and the problem goes away

Reply:
why would it require a revbump, there is no potential for breakage? i think a failing build is even worse UX. 😊

Reply:
`RDEPEND` changes always need a revbump or they do not end up in VDB, so existing stable ebuild users that you try to protect here in fact won't be stopped from updating exiv2.

Reply:
> I think a failing build is even worse UX.

You are faced with the option of receiving bugs about failing build because of an unsupported setup, or bugs about Portage conflicts related to your package, pick your poison.

Reply:
ah, hadn't thought of that, thanks. dropped the commit.